.github/actions/setup-miniconda/action.yml

```diff
@@ -27,60 +27,6 @@ inputs:
 runs:
   using: composite
   steps:
```

Comment:
Don't you want to invoke the new action here?

Reply:
Doing so would require whoever using `setup-miniconda` to checkout `test-infra` repo because it uses the other actions.  GitHub action is not smart enough to figure this out recursively I think.  AFAIK, it seems clearer to have a clean up step, and a setup miniconda step in separate actions, so that I can use the former in different place out side of setup miniconda as well.  So I decide to remove it from setup-miniconda